.github/workflows/canary.yaml

```diff
@@ -0,0 +1,54 @@
+name: Canary build
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+
+jobs:
+  build-binaries:
+    name: build binaries
```

Comment:
For consistency, name should be capitalized, Build binaries

Reply:
Done